templates/consul_catalog.tmpl

```diff
@@ -1,12 +1,33 @@
-[backends]{{range .Nodes}}
+[backends]
+{{range .Nodes}}
     [backends.backend-{{getBackend .}}.servers.server-{{.Node.Node | replace "." "-"}}-{{.Service.Port}}]
-    url = "http://{{.Node.Address}}:{{.Service.Port}}"
+    	url = "http://{{.Node.Address}}:{{.Service.Port}}"
```

Comment:
Would it be easy to add `traefik.protocol` here as a way to support HTTPS to backends from consul?